spago.dhall

```diff
@@ -0,0 +1,19 @@
+{ sources = [ "src/**/*.purs", "test/**/*.purs" ]
+, name = "dom-simple"
```

Comment:
Should this be `purescript-milkis`?

Reply:
Yes, seems I mismatched branches, I'll create a new PR.